src/components/Map/Map/Map.js

```diff
@@ -70,12 +71,35 @@ export const Map = React.forwardRef((props, ref) => {
         }
     }, [defaultArea, mapRef, markers]);
 
-    useEffect(fitBounds);
+    const manageDefaultHighlight = React.useCallback(() => {
+        if (mapRef.current && mapRef.current.state.map) {
+            const { map } = mapRef.current.state;
+            if (defaultHighlight && !markers.length && !highlightFeatures) {
```

Comment:
This condition looks like we only can render highlighted area if there are no markers passed.

It might be an JobFeed requirement, but then I think it should be handled on the application level.

And if it is technically possible to render both markers and areas, I don't see why we should not allow to do in on the component level.

Reply:
These are the requirements in the ticket. @carlobernardini ?

Reply:
> These are the requirements in the ticket

Sure, and this is product level requirements: it can be implemented either on OneUI level or on the application level. My point is just to remind that AFAIR once we decided to keep OneUI components as simple as possible without "business" logic in it.

Imagine.. Years from now... There is a developer trying to use this `<Map />` component. They want to render a geojson data, they copy an example from storybook (it has few markers in place), they add `defaultHighlighting`,  check the results and see only few markers instead of highlighted area. That's confusing... :) Expected behaviour would be to see both things and instantly realise that "ah.. I also passed markers, I should remove them".

But yeah, I wasn't involved into interface design phase.. Since 2 pair of eyes checked this idea, I guess we can live with it then :)

Reply:
I do agree with your reasoning @antipin. But since @eszthoff already put quite some effort in this PR let's keep the API as it is. And later, when the scenario of highlighting multiple areas or mixing areas and markers presents itself, add an additional prop e.g. `polygons` and either keep `defaultHighlighting` for the current use case - or we 
deprecate it if you eventually decide to move the business logic into the application (which I'd then prefer to keep the API concise).